include/rocksdb/options.h

```diff
@@ -57,6 +57,21 @@ class FileSystem;
 struct Options;
 struct DbPath;
 
+// To indicate the file types
+enum FileType {
```

Comment:
Since this is not public, can you add a brief comment as to what all of these mean?  

Also, should there be a kUnknownFile for types that we do not know about?

Reply:
Sure. Totally agree, some of the name does not directly indicate the file type. I will add the comments. Since I move it from file_name.h to options.h, I prefer not to add kUnknownFile at this moment. In the future, if we have the use case, we can add it easily.